app/schema.graphql

```diff
@@ -110,6 +110,7 @@ input ChatCompletionInput {
   tools: [JSON!]
   apiKey: String = null
   template: TemplateOptions
+  promptId: GlobalID = null
```

Comment:
Do we want to also add this to the nonStreaming mutation? That one does not carry over prompt id